src/global_heap.cc

```diff
@@ -416,9 +420,9 @@ void GlobalHeap::meshAllSizeClassesLocked() {
     totalMeshCount += meshSizeClassLocked(sizeClass, MergeSets, Left, Right);
   }
 
-  madvise(&Left, sizeof(Left), MADV_DONTNEED);
-  madvise(&Right, sizeof(Right), MADV_DONTNEED);
-  madvise(&MergeSets, sizeof(MergeSets), MADV_DONTNEED);
```

Comment:
is your thought that these just aren't big enough chunks of memory to free?

Reply:
I think it is a big chance need to use this 3 buffer again at the next meshing, so no need to free it at all.